specification/billing/resource-manager/Microsoft.Billing/preview/2018-03-01-preview/billing.json

```diff
@@ -402,7 +443,7 @@
           "$ref": "#/definitions/EnrollmentAccountProperties"
         }
       }
-    },
+    },    
```

Comment:
Trim?

Reply:
done

Reply:
:+1:  looks like you replaced the next line indentation with a tab though :wink:

Reply:
I don't see any tab in notepadd ++.  Can you please approve it else it won't get published as ARM api.